config/300-channel.yaml

```diff
@@ -56,15 +56,11 @@ spec:
             secret:
               type: object
               # TODO: document we default.
-              description: "Credential to use to manage Cloud Pub/Sub. The value of the secret entry must be a service account key in the JSON format (see https://cloud.google.com/iam/docs/creating-managing-service-account-keys)."
+              description: "Credential to use to manage Cloud Pub/Sub. The value of the secret entry must be a service account key in the JSON format (see https://cloud.google.com/iam/docs/creating-managing-service-account-keys). Defaults to secret.name of 'google-cloud-key' and secret.key of 'key.json'."
             project:
               type: string
               # TODO: document we default.
```

Comment:
might as well document now?

Reply:
I filed an issue as a follow because it applies to all resouces